lib/matplotlib/rcsetup.py

```diff
@@ -818,6 +836,12 @@ def validate_cycler(s):
         norm_prop = _prop_aliases.get(prop, prop)
         cycler_inst.change_key(prop, norm_prop)
 
+    for key, vals in cycler_inst.by_key().items():
+        _prop_validators[key](vals)
+
+    if 'color' not in cycler_inst.keys:
+        cycler_inst = cycler_inst * cycler('color', ['k'])
```

Comment:
I honestly do not agree with this at all. This will cause unexpected behavior, which we noted already because the cycler will advance when the user did not expect it to. Why can't CN just fall back to the default linecolor (which is always defined) if the property cycle doesn't have color?

What I would agree with is if the cycler has *nothing*, then it should fall back to something. What that something is, I am not sure (hard to reference an rcParam while in the middle of processing them...).